addons/website_forum/models/forum.py

```diff
@@ -37,16 +37,11 @@ def _get_default_faq(self):
         help='Questions mode: only one answer allowed\n Discussions mode: multiple answers allowed')
     active = fields.Boolean(default=True)
     faq = fields.Html('Guidelines', default=_get_default_faq, translate=html_translate, sanitize=False)
-    description = fields.Text(
-        'Description',
-        translate=True,
-        default=lambda s: _('This community is for professionals and enthusiasts of our products and services. '
-                            'Share and discuss the best content and new marketing ideas, '
-                            'build your professional profile and become a better marketer together.'))
+    description = fields.Text('Description', translate=True)
```

Comment:
why do you want remove the default description ? it is a request merged a few month ago that ask to move from the demo to the default data.

Reply:
@sbuhl ? opinion?

Reply:
Several reasons. The description it's NOT in the sidebar anymore, it's on the index page where all the forums (and courses) are listed.
Just imagine a list of forums/courses, all of them with the same description. The user had to manually remove/edit them, because it was added by default.

Reply:
Can we keep the default desc. please.

Reply:
This is what you'll get with the description as default.
@sbuhl , can you confirm please? 

![Screenshot 2019-08-19 at 15 51 37](https://user-images.githubusercontent.com/8558351/63270879-441c4900-c299-11e9-8674-a099c35dad7b.png)

Reply:
Ha sorry, didn't saw your previous message before mine. Of course if you put the description there, it doesn't look good.